### DISCUSSION TITLE
Error while copying directories include spaces in name in amazon S3 bucket

### DISCUSSION THREAD

**Discussion by BurhanNoori** [aws/aws-sdk-java#3034] [Q&A]
Hi team, 

I am able to copy the directories that does not include spaces in its name whereas not able to do so with names include spaces in S3. How can I achieve this?

**Comment by debora-ito** [comment]
@BurhanNoori can you share a code sample we can use to reproduce the error?

Please also provide:
- The SDK version you're using
- Stacktrace and logs with the error
- OS and version

**Reply by BurhanNoori** [reply]
@debora-ito ,
Sorry cannot share the code but I can provide you the following details:
 sdk version - aws-java-sdk-bundle-1.11.563
 OS - Linux (all versions)
